### PR TITLE
Parallel commands   rock

### PR DESCRIPTION
**Pull request by romallan**
-ShooterPID: sets shooter to certain rpm and ends but continues spinning until the robot does not contain note (Does not run feeder)
-Feeder: runs feeder to parameter and sets shooter and feeder to zero once robot doesn't contain note
-PrepareShooter: Runs PivotPID and ShooterPID in a parallel command group to raise arm and run the shooter
-ShooterSequence: runs sequence command group of alignScore then PrepareShooter then feeder then PivotPID to intake position
-Shooter is supposed to be the same as shooterSequence minus align and pivot down

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -64,9 +62,9 @@ public RobotContainer() {
     
     // Pathplanner Commands for use in auto. Name is what you type into pathplanner,
     // and the commands are "borrowed" from the controller
-    NamedCommands.registerCommand("Score Close", new ScoreGoal(ManipulatorConstants.PIVOT_CLOSE_SCORE));
-    NamedCommands.registerCommand("Score Far", new ScoreGoal(ManipulatorConstants.PIVOT_FAR_SCORE));
-    NamedCommands.registerCommand("Score Amp", new AmpTrapScore(ManipulatorConstants.PIVOT_AMP_GOAL));
+    NamedCommands.registerCommand("Score Close", new Shooter(ManipulatorConstants.SCORE_SIMPLE_RPM, ManipulatorConstants.PIVOT_CLOSE_SCORE));
+    NamedCommands.registerCommand("Score Far", new Shooter(ManipulatorConstants.SCORE_SIMPLE_RPM,ManipulatorConstants.PIVOT_FAR_SCORE));
```
Comment (minhd-vu) [inline src/main/java/frc/robot/RobotContainer.java:66]:
```suggestion
    NamedCommands.registerCommand("Score Far", new Shooter(ManipulatorConstants.SCORE_SIMPLE_RPM, ManipulatorConstants.PIVOT_FAR_SCORE));
```

File: src/main/java/frc/robot/commands/manipulator/commandgroup/ShooterSequence.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import frc.robot.Constants.ManipulatorConstants;
+import frc.robot.commands.manipulator.pivot.PivotPID;
+import frc.robot.commands.manipulator.shooter.Feeder;
+import frc.robot.commands.vision.AlignScore;
+
+public class ShooterSequence extends SequentialCommandGroup{
+    
+    public ShooterSequence(double shootSpeed, double pivotGoal) {
+    addCommands(
+        new AlignScore(),
```
Comment (zabzzz) [inline src/main/java/frc/robot/commands/manipulator/commandgroup/ShooterSequence.java:13]: We need a better way to implement this, this command will never end, thus we never reach the parallel command group to spin up the wheels and to up the pivot.

Comment (romallan) [reply src/main/java/frc/robot/commands/manipulator/commandgroup/ShooterSequence.java:13]: I kind of put this here as a place holder honestly. Once we touch up align score we can add it to the shooter sequence.

File: src/main/java/frc/robot/commands/manipulator/commandgroup/ShooterParallel.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.manipulator.commandgroup;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.commands.manipulator.pivot.PivotPID;
+import frc.robot.commands.manipulator.shooter.ShooterPID;
+
+public class ShooterParallel extends ParallelCommandGroup {
```
Comment (minhd-vu) [inline src/main/java/frc/robot/commands/manipulator/commandgroup/ShooterParallel.java:11]: @zabzzz do you think a
> A ParallelDeadlineGroup ([Java](https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/ParallelDeadlineGroup.html), [C++](https://first.wpi.edu/wpilib/allwpilib/docs/release/cpp/classfrc2_1_1ParallelDeadlineGroup.html)) also runs a set of commands concurrently. However, the deadline group ends when a specific command (the “deadline”) ends, interrupting all other commands in the group that are still running at that point.

Would work?

**Pull request closed without merging**